src/main/kotlin/org/rust/lang/core/psi/ext/RsElement.kt

```diff
@@ -145,3 +143,10 @@ fun RsElement.findInScope(name: String): PsiElement? {
     }
     return resolved
 }
+
+
+val RsElement.isLocal: Boolean
```

Comment:
Looks like a something very specific to completion sorting. Maybe move it to LookupElements.kt and make private?